pages.nl/linux/apt.md

```diff
@@ -0,0 +1,38 @@
+# apt
+
+> Hulpprogramma voor pakketbeheer voor op Debian gebaseerde distributies.
+> Aanbevolen vervanging voor `apt-get` bij interactief gebruik in Ubuntu versie 16.04 en later.
+> Voor gelijkwaardige commando's in andere pakket managers, zie <https://wiki.archlinux.org/title/Pacman/Rosetta>.
+> Meer informatie: <https://manpages.debian.org/latest/apt/apt.8.html>.
+
+- Werk de lijst van beschikbare pakketten en versies bij (het wordt aanbevolen dit uit te voeren voor elk ander `apt` commando):
+
+`sudo apt update`
+
+- Zoek naar een bepaald pakket:
+
+`apt search {{package}}`
+
+- Toon informatie over een pakket:
+
+`apt show {{package}}`
+
+- Installeer of werk het pakket bij naar de laatste beschikbare versie:
+
+`sudo apt install {{package}}`
```

Comment:
```suggestion
- Installeer specifieke pakketten of werk ze bij naar de nieuwste beschikbare versies:

`sudo apt install {{pakket1 pakket2 ...}}`
```

The same issue as above.
Note: translation source: Install specific packages, or update them to the latest available versions.

Reply:
Thanks for the review! Fixed in the latest commit.